lib/engine/g_18_tn/share_pool.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require_relative '../share_pool'
+
+module Engine
+  module G18TN
+    class SharePool < SharePool
+      def buy_shares(entity, shares, exchange: nil, exchange_price: nil)
+        super
+
+        return if @game.turn != 1
```

Comment:
return if !@game.turn.one? || (lnr != @game.company(..)
return unless lnr.owner

lnr.close!

Reply:
Fixed

Reply:
this doesn't look fixed

Reply:
Ah, another try has been made.